packages/view/backend/sirius-components-view-diagram-customnodes-edit/src/main/java/org/eclipse/sirius/components/view/diagram/customnodes/provider/EllipseNodeStyleDescriptionItemProvider.java

```diff
@@ -109,11 +109,11 @@ protected void addBorderLineStylePropertyDescriptor(Object object) {
     /**
      * This returns EllipseNodeStyleDescription.gif. <!-- begin-user-doc --> <!-- end-user-doc -->
      *
-     * @generated NOT
+     * @generated
      */
     @Override
     public Object getImage(Object object) {
-        return this.overlayImage(object, this.getResourceLocator().getImage("full/obj16/EllipseNodeStyle.svg"));
+        return this.overlayImage(object, this.getResourceLocator().getImage("full/obj16/EllipseNodeStyleDescription"));
```

Comment:
`@generated NOT` + `EllipseNodeStyle.svg` lost.

Reply:
Done